expenses-manager-backend/src/main/java/com/endava/expensesmanager/controller/ExpenseController.java

```diff
@@ -60,4 +60,13 @@ public ResponseEntity<BigDecimal> getTotalAmountByDateBetween(@RequestParam int
         BigDecimal total = expenseService.getTotalAmountByDateBetween(userId, startDate, endDate);
         return ResponseEntity.ok(total);
     }
+
+    @GetMapping("/user/{userId}/")
+    public ResponseEntity<List<ExpenseDto>> getExpensesPage(@PathVariable int userId, @RequestParam(required = false) LocalDateTime startDate, @RequestParam(required = false) LocalDateTime endDate,
```

Comment:
I would suggest that each parameter be on a new line, to make the code easier to read